.github/ISSUE_TEMPLATE/tc2_bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Team Comtress bug report
+about: A bug that occurs in Team Comtress
+title:
+labels: bug, tc2, needs triage
+assignees:
+
+---
+
+### Description
+<!--
+A clear and concise description of what the bug is.
+-->
+
+### Checklist
+<!-- You do not have to answer "yes" to all of these to open an issue. -->
+- [ ] None of the *open or closed* issues document this problem.
+- [ ] This is a bug, not a new idea.
+- [ ] This is obviously a bug, not something that people commonly disagree on.
+- [ ] This bug is in the code of the game, not in the textures, sounds, or other resources.
+- [ ] This issue occurs on the lastest vanilla TF2 update.
+- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
+- [ ] This issue occurs with DirectX level 100.
+- [ ] This issue occurs on the latest Team Comtress release, or the `master` branch.
+- [ ] This issue occurs when using no custom content.
+- [ ] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
+- [ ] This issue occurs when using a server started with the `map` command from the game.
+- [ ] This issue occurs when using a dedicated server.
+- [ ] This bug is in the internal procedures or tooling (e.g. the build process).
```

Comment:
does this last line have to be here for TF2 issues?

Reply:
Yes, because the issue can affect both TF2 and TC2